src/QMCWaveFunctions/Jastrow/GaussianFunctor.h

```diff
@@ -9,22 +9,27 @@
 // File created by: Brett Van Der Goetz, bvdg@berkeley.edu, University of California at Berkeley
 //////////////////////////////////////////////////////////////////////////////////////
 
-#ifndef QMCPLUSPLUS_COUNTING_FUNCTOR_H
-#define QMCPLUSPLUS_COUNTING_FUNCTOR_H
+#ifndef QMCPLUSPLUS_GAUSSIAN_FUNCTOR_H
+#define QMCPLUSPLUS_GAUSSIAN_FUNCTOR_H
 
 #include "OhmmsData/AttributeSet.h"
+#include "Optimize/VariableSet.h"
 #include <array>
 
 namespace qmcplusplus
 {
-// doesn't inherit from OptimizableFunctorBase since this is a function of the entire position vector
-template<class T>
-class GaussianFunctor : public QMCTraits
+
```

Comment:
Need some comment for this class documenting all the use cases and limitations. The comment you deleted seems like useful info.

This is GaussianFunctor can not be used directly in J2 to replace the BsplineFunctor, having similar name is confusing. Please find a name more verbose.

Reply:
I've readded that comment and changed GaussianFunctor to CountingGaussian and NormalizedGaussianRegion to CountingGaussianRegion for consistency and clarity.